wagtail/contrib/settings/tests/generic/test_model.py

```diff
@@ -50,7 +50,7 @@ def test_request_or_site_with_request_result_caching(self):
 
                 # only the first lookup should result in a query
                 with self.assertNumQueries(1):
-                    for i in range(4):
+                    for _ in range(4):
```

Comment:
We generally avoid using `_` as a placeholder variable, as it interferes with the common Django pattern of using it as an alias for `gettext` / `gettext_lazy`.

Reply:
@gasman, thank you for your response! You are right; I will revert these changes. Although, I believe that in `wagtail/test/benchmark.py` it's not gonna interfere with Django translation aliases.